Core/MIPS/ARM64/Arm64Jit.cpp

```diff
@@ -519,101 +519,23 @@ void Arm64Jit::WriteDownCountR(ARM64Reg reg) {
 void Arm64Jit::RestoreRoundingMode(bool force) {
 	// If the game has never set an interesting rounding mode, we can safely skip this.
 	if (g_Config.bSetRoundingMode && (force || !g_Config.bForceFlushToZero || js.hasSetRounding)) {
-		MRS(SCRATCH2_64, FIELD_FPCR);
-		// Assume we're always in round-to-nearest mode beforehand.
-		// Also on ARM, we're always in flush-to-zero in C++, so stay that way.
-		if (!g_Config.bForceFlushToZero) {
-			ORRI2R(SCRATCH2, SCRATCH2, 4 << 22);
-		}
-		ANDI2R(SCRATCH2, SCRATCH2, ~(3 << 22));
-		_MSR(FIELD_FPCR, SCRATCH2_64);
+		QuickCallFunction(SCRATCH2_64, restoreRoundingMode);
 	}
 }
 
 void Arm64Jit::ApplyRoundingMode(bool force) {
-	// NOTE: Must not destroy SCRATCH1.
+	// NOTE: Must not destroy SCRATCH1. TODO: Why?
```

Comment:
Because callers assume it won't:
https://github.com/hrydgard/ppsspp/blob/0ffffb41bd7be46bb0050813f15a7aba80f6afd6/Core/MIPS/ARM/ArmJit.cpp#L430
https://github.com/hrydgard/ppsspp/blob/0ffffb41bd7be46bb0050813f15a7aba80f6afd6/Core/MIPS/ARM/ArmRegCache.h#L29

But this comment from arm32 may not apply to arm64...

-[Unknown]